api/transactions/types.go

```diff
@@ -220,3 +220,8 @@ func convertReceipt(txReceipt *tx.Receipt, header *block.Header, tx *tx.Transact
 	}
 	return receipt, nil
 }
+
+// TxSendResult is the response to the Send Tx method
+type TxSendResult struct {
```

Comment:
```suggestion
type SendTxResult struct {
```
Following the name convention of `GetCodeResult`.

Reply:
Fixed !